inference/server/alembic/versions/2023_02_26_1328-a2865908a537_add_worker_compliance_checks_to_db.py

```diff
@@ -0,0 +1,52 @@
+"""Add worker compliance checks to db
+
+Revision ID: a2865908a537
+Revises: e7a4bffb424c
+Create Date: 2023-02-26 13:28:25.181340
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a2865908a537"
+down_revision = "e7a4bffb424c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "worker_compliance_check",
+        sa.Column("id", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("worker_id", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("start_time", sa.DateTime(), nullable=False),
+        sa.Column("end_time", sa.DateTime(), nullable=True),
+        sa.Column("responded", sa.Boolean(), nullable=False),
+        sa.Column("error", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+        sa.Column("passed", sa.Boolean(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["worker_id"],
+            ["worker.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_worker_compliance_check_worker_id"), "worker_compliance_check", ["worker_id"], unique=False
+    )
+    op.alter_column("chat", "user_id", existing_type=sa.VARCHAR(), nullable=False)
```

Comment:
I am not sure why Alembic generated this as I don't think this PR should have made any change affecting this. If anyone can confirm what might have caused this, we might want to remove it?

Reply:
if you look at the downgrade, then it looks like the type is kept exactly the same. I don't think it's detrimental to just leave it in